m/mesa/stone.yaml

```diff
@@ -4,11 +4,11 @@
 # SPDX-License-Identifier: MPL-2.0
 #
 name        : mesa
-version     : 24.3.4
-release     : 30
+version     : 25.0.1
```

Comment:
Can you add a note here that `mesa-clc` should be kept in sync with mesa version-wise?

Reply:
Why not find a way in tooling to allow 64bit split packages for 32bit base packages so all is in one spec?

Reply:
So you want to make it so that we build mesa in one recipe and cross-compile within the same recipe? It's certainly possible but I suspect that it'll be more annoying to setup than just having a split out mesa-clc package.